### PR TITLE
CIV-0000_fix-vulnerabilities: fixed vulnerabilities

### PR DESCRIPTION
**Pull request by srochani**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by rishikrsharma — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -210,7 +210,10 @@
     "nunjucks": "3.2.4",
     "xml2js": "0.5.0",
     "cookiejar": "2.1.4",
-    "ajv": "6.12.3"
+    "ajv": "6.12.3",
+    "express": "4.19.2",
```
Comment (linusnorton) [inline package.json:214]: duplicate

File: package.json [hunk 2]
```diff
@@ -210,7 +210,10 @@
...
+    "tar": "6.2.1",
```
[the remark this answers is not in the document]
Comment (srochani) [reply package.json:215]: Thanks, here I'm not adding the new dependencies, tar is already being use, just update to use latest version

File: package.json (hunk 1 above)
Comment (srochani) [reply package.json:214]: Thanks, removed duplicate

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (linusnorton) [reply package.json:214]: please use [resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) instead

[the remark this answers is not in the document]
Comment (srochani) [reply package.json:214]: its already in https://github.com/hmcts/cmc-citizen-frontend/blob/a66bb0e353e3286d130158d7888da9c0cecd0342/package.json#L213

**Review by linusnorton — APPROVED** [review]

**Pull request merged**